cmake/CodeCoverage.cmake

```diff
@@ -71,7 +71,7 @@
 # - Added the option for users to set the GCOVR_ADDITIONAL_ARGS variable to supply additional
 #   flags to the gcovr command
 #
-# 2020-05-04, Mihchael Davis
+# 2020-05-04, Michael Davis
```

Comment:
I'm fairly certain this is the correct person's name, but I'm not really sure where this code comes from -- I'd love to fix the source material.

Reply:
The original is here https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake

Reply:
Thanks. I posted https://github.com/bilke/cmake-modules/pull/90/files#r1878500171 to upstream this fix.